### PR TITLE
Fix HMD.mountedChanged() API signal

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11413/HighFidelity-Beta-PR13642-5e62424.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11413/HighFidelity-Beta-PR13642-5e62424.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11413/HighFidelity-Beta-Interface-PR13642-5e62424.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11413/HighFidelity-Beta-PR13642-5e62424.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11413/HighFidelity-Beta-Interface-PR13642-5e62424.dmg

File: libraries/display-plugins/src/display-plugins/hmd/HmdDisplayPlugin.h [hunk 1]
```diff
@@ -45,6 +46,9 @@ class HmdDisplayPlugin : public OpenGLDisplayPlugin {
 
     virtual bool onDisplayTextureReset() override { _clearPreviewFlag = true; return true; };
 
+signals:
+    void isHmdMountedChanged();
```
Comment (jherico) [inline libraries/display-plugins/src/display-plugins/hmd/HmdDisplayPlugin.h]: technically this signal should be `hmdMountedChanged`

File: plugins/oculus/src/OculusBaseDisplayPlugin.h [hunk 2]
```diff
@@ -44,4 +44,5 @@ class OculusBaseDisplayPlugin : public HmdDisplayPlugin {
     ovrLayerEyeFov _sceneLayer;
     ovrViewScaleDesc _viewScaleDesc;
     // ovrLayerEyeFovDepth _depthLayer;
+    bool _isHmdMounted { false };
```
Comment (jherico) [inline plugins/oculus/src/OculusBaseDisplayPlugin.h]: similarly this should be `_hmdMounted` not `_isHmdMounted`.  the verb prefix is reserved for getters and setters, with `get` or `is` being used depending on whether the property is boolean or not.

**Comment by qlimnavarro** [comment]
Currently Testing

File: plugins/oculus/src/OculusBaseDisplayPlugin.h (hunk 2 above)
Comment (ctrlaltdavid) [reply plugins/oculus/src/OculusBaseDisplayPlugin.h]: Thanks

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11427/HighFidelity-Beta-PR13642-a3e677e.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11427/HighFidelity-Beta-PR13642-a3e677e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11427/HighFidelity-Beta-Interface-PR13642-a3e677e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11427/HighFidelity-Beta-PR13642-a3e677e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11427/HighFidelity-Beta-Interface-PR13642-a3e677e.dmg

**Comment by qlihifitest** [comment]
Currently Testing

**Comment by qlihifitest** [comment]
Passes on Vive and Oculus. Script reported everything correctly

@qliacoombs @Qliemillar @conklin94122

**Comment by huffman** [comment]
@jherico Do these changes look correct to you?

**Pull request merged**